shell.nix

```diff
@@ -0,0 +1,10 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+pkgs.mkShell {
+  buildInputs = with pkgs; [
+    go goimports gopls
```

Comment:
Missing git, which is invoked explicitly in some build paths to embed release info.